src/runtime/mod.rs

```diff
@@ -251,34 +246,6 @@ impl Runtime {
         Builder::new().build()
     }
 
-    #[deprecated(since = "0.1.5", note = "use `reactor` instead")]
-    #[doc(hidden)]
-    pub fn handle(&self) -> &Handle {
-        self.reactor()
-    }
-
-    /// Return a reference to the reactor handle for this runtime instance.
-    ///
-    /// The returned handle reference can be cloned in order to get an owned
-    /// value of the handle. This handle can be used to initialize I/O resources
-    /// (like TCP or UDP sockets) that will not be used on the runtime.
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// use tokio::runtime::Runtime;
-    ///
-    /// let rt = Runtime::new()
-    ///     .unwrap();
-    ///
-    /// let reactor_handle = rt.reactor().clone();
-    ///
-    /// // use `reactor_handle`
-    /// ```
-    pub fn reactor(&self) -> &Handle {
-        self.inner().reactor.handle()
-    }
```

Comment:
Not sure what to do about `reactor()` and `handle()` other than to remove them. Perhaps we could still keep the background reactor, but would it be worth it?